README.md

```diff
@@ -0,0 +1,147 @@
+otel-cli exec --service my-service --name "curl google" curl https://google.com
+
+# a more visually interesting example trace
+carrier=$(mktemp)
+sockdir=$(mktemp -d)
+otel-cli span background \
+   --service "otel-cli-example" \
+   --name "otel-cli-example background span" \
+   --tp-print \
+   --tp-carrier $carrier \
+   --sockdir $sockdir &
+sleep 0.1 # give the background server just a few ms to start up
+otel-cli span event --name "cool thing" --attrs "foo=bar" --sockdir $sockdir
+otel-cli exec --service "otel-cli-example" --name "curl google" --tp-carrier $carrier curl https://google.com
+sleep 0.1
+otel-cli exec --service "otel-cli-example" --name "curl google" --tp-carrier $carrier curl https://google.com
+sleep 0.1
+otel-cli span event --name "another cool thing\!" --attrs "foo=bar" --sockdir $sockdir
+otel-cli span end --sockdir $sockdir
+```
+
+![otel-cli-example](https://user-images.githubusercontent.com/56372758/217082956-23c60f2d-f882-4c78-a205-f744596fac21.png)
+
+## Why does this exist?
+
+When we send OpenTelemetry telemetry to a tracing vendor we expect to be able to visualize our
+data, but when working locally our experience often looks more like this:
+
+```
+{
+	"Name": "Poll",
+	"SpanContext": {
+		"TraceID": "4d7f165e90111f4fc9003d5bbf7aca81",
+		"SpanID": "1f7a655a9b7a6f4b",
+		"TraceFlags": "01",
+		"TraceState": "",
+		"Remote": false
+	},
+	"Parent": {
+		"TraceID": "4d7f165e90111f4fc9003d5bbf7aca81",
+		"SpanID": "d1b8f7f96ad9d1a0",
+		"TraceFlags": "01",
+		"TraceState": "",
+		"Remote": false
+	},
+  ...
+```
+
+You can use [Jaeger's all-in-one](https://www.jaegertracing.io/docs/1.41/deployment/#all-in-one)
+distribution, but this requires quite a bit of additional knowledge for the end-user of docker,
+configuration options, and the user experience is not focused around "show me the data I just
+emitted".
+
+The goal for `otel-desktop-viewer` is to allow a user to install it with one command, require
+minimal configuration and additional tooling, and be as approachable as possible for developers
+at all levels of experience.
+
+
+## Implementation
+
+The CLI is implemented in Go building on top of the OpenTelemetry Collector. A custom
+`desktop` exporter is registered that:
+
+- collects trace data in memory
+- exposes this trace data via an HTTP API
+- serves a static React app that renders the collected traces
+
+All of the static web assets are built into the final binary using the [go:embed](https://blog.jetbrains.com/go/2021/06/09/how-to-use-go-embed-in-go-1-16/)
+directive so that the binary is self-contained and relocatable.
+
+## What's with the axolotl??
+
+Her name is Lulu Axol'Otel, she is very pink, and I love her.
+
+More seriously, I like to give my [side projects](https://github.com/CtrlSpice/bumblebee-consolematch) an 
+[animal theme](https://github.com/CtrlSpice/yak-vs-yak) to add a little aesthetic
+interest on what otherwise might be fairly plain applications.
```

Comment:
Let's add a section at the end for License a la otel-cli https://github.com/equinix-labs/otel-cli#license